modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/TypeScriptClientCodegenTest.java

```diff
@@ -53,7 +53,11 @@ public void testComposedSchemasImportTypesIndividually() {
         codegen.setOpenAPI(openApi);
         PathItem path = openApi.getPaths().get("/pets");
         CodegenOperation operation = codegen.fromOperation("/pets", "patch", path.getPatch(), path.getServers());
-        Assert.assertEquals(operation.imports, Sets.newHashSet("Cat", "Dog"));
+        // TODO revise the commented test below as oneOf is no longer defined inline
+        //but instead defined using $ref with the new inline model resolver in 6.x
+        //Assert.assertEquals(operation.imports, Sets.newHashSet("Cat", "Dog"));
+        Assert.assertEquals(operation.imports, Sets.newHashSet("InlineRequest"));
```

Comment:
@TiFu FYI. We've updated the inlined resolver to create a separate model/schema for all inline schemas(including oneOf/allOf) in the request bodies.